lib/sensu-plugin/metric/cli.rb

```diff
@@ -19,13 +18,12 @@ def output(obj=nil)
 
         class Graphite < Sensu::Plugin::CLI
           def output(*args)
-            unless args.empty?
-              if args[0].is_a?(Exception) || args[1].nil?
-                puts args[0].to_s
-              else
-                args[2] ||= Time.now.to_i
-                puts args[0..2].join("\s")
-              end
+            return '' if args.empty?
```

Comment:
I think this is technically behavior changing before it would have been nil now its an empty string.